hudi-hive/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -53,30 +53,44 @@
 public class HiveSyncTool {
 
   private static final Logger LOG = LogManager.getLogger(HiveSyncTool.class);
-  private final HoodieHiveClient hoodieHiveClient;
-  public static final String SUFFIX_REALTIME_TABLE = "_rt";
+  public static final String SUFFIX_SNAPSHOT_TABLE = "_rt";
+  public static final String SUFFIX_READ_OPTIMIZED_TABLE = "_ro";
```

Comment:
Not sure if it's the best user-experience to augment the table name with a postfix for all tables..